### PR TITLE
fix: add steps in pipeline to create aws creds file

### PR DESCRIPTION
**Pull request by maghirardelli**
Issue #, if available:

Description of changes:
Trying to fix the pipelines so that the new Serverless V3 syntax behaves with the aws credentials configuration within the pipeline environments.
This is option 2 where we are adding steps to manually create `~/.aws/credentials` files in the pipeline environments so that they can use the `default` profile within Serverless V3. The GH Actions pipeline step has been tested and works. The CodePipeline step has not been tested.

Checklist:

AS review ticket id:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: main/cicd/cicd-pipeline/config/buildspec/buildspec.yml [hunk 1]
```diff
@@ -21,6 +21,16 @@ phases:
       - echo "Downloading environment config"
       - ./scripts/download-env-config.sh "$DEPLOYMENT_BUCKET" "$ENV_NAME"
       - printf "\n\n"
+      - echo "Creating ~/.aws/credentials"
```
Comment (nguyen102) [inline main/cicd/cicd-pipeline/config/buildspec/buildspec.yml:24]: I think you need to add something similar for `buildspec-int-test.yml`

File: .github/workflows/deploy-integ-appstream-egress.yml [hunk 2]
```diff
@@ -36,6 +36,10 @@ jobs:
           go-version: 1.13
```
Comment (nguyen102) [inline .github/workflows/deploy-integ-appstream-egress.yml:36]: On line 33 and the other lines that specify node-version: 12, let's use node v16.
https://github.com/actions/setup-node

File: main/cicd/cicd-pipeline/config/buildspec/buildspec.yml (hunk 1 above)
Comment (maghirardelli) [reply main/cicd/cicd-pipeline/config/buildspec/buildspec.yml:24]: There are no `sls` commands in the script that build runs([run-integration-tests.sh](https://github.com/awslabs/service-workbench-on-aws/blob/mainline/scripts/run-integration-tests.sh)). Why do you think we need to do something similar?

File: .github/workflows/deploy-integ-appstream-egress.yml (hunk 2 above)
Comment (maghirardelli) [reply .github/workflows/deploy-integ-appstream-egress.yml:36]: Updated!

File: main/cicd/cicd-pipeline/config/buildspec/buildspec.yml (hunk 1 above)
Comment (nguyen102) [reply main/cicd/cicd-pipeline/config/buildspec/buildspec.yml:24]: nevermind, you're right. Resolving this comment.

**Review by nguyen102 — APPROVED** [review]

**Pull request merged**